robyn/import_vars.py

```diff
@@ -0,0 +1,27 @@
+import os 
+from definitions import CONFIG_PATH
+
+# parse the configuration file returning a list of tuples (key, value) containing the environment variables
+def parser():
+    """Parse the configuration file"""
+    with open(CONFIG_PATH, 'r') as f:
+        for line in f:
+            if line.startswith('#'):
+                continue
+            yield line.strip().split('=')  
+
+
+# check for the environment variables set in cli and if not set them
+
+def load_vars(variables = parser()):
+    """Main function"""
+    for var in variables:
+        if var[0] in os.environ:
```

Comment:
Do you think we would want to override the existing environment variables or not?

Reply:
I thought the priority is for environment variables that are set in the terminal or manually by the user not the ones that are in the config file so i made the code check for variables that already exist and not update those

Reply:
Okay, makes sense. ✨